bug fix - when the first section has a background-video, the vido do …

alexbatista opened the pull request:
correction based on the issue: 
Background video does not play on the first slide loaded #1314

Delapouite commented:
Thanks for your contribution. 

Some part of your PR (spacing, indents…) do not match the overall coding style of reveal. Could you please fix these details please? Thanks.

alexbatista commented:
Hi Delapouite, i updated the code. I removed the extras lines, but i think that indentation is right, no? If no, how many spaces the reveal.js adopt? Appears adjusted on my machine. 

Thanks!

hakimel commented:
Thanks for the fix! Your commit has been merged into the dev branch, in the future please submit all pull requests directly towards the dev branch.

I made a few minor tweaks in 3d33d27
